app/forms/validation/StringConstraints.scala

```diff
@@ -25,7 +25,7 @@ object StringConstraints {
   val numericalCharacters = """[0-9.]*"""
   val monetaryRegex = """\d+|\d*\.\d{1,2}"""
   val alphabetsWithSpaceRegex = """^([a-zA-Z])+([a-zA-Z ])*$"""
-  val policyNumberRegex = """^$|^(?=.*[0-9])(?=.*[a-zA-Z])([a-zA-Z0-9\/-]+)$"""
+  val policyNumberRegex = """^[0-9a-zA-Z{À-˿’}\- _&`():.'^]{1,90}$"""
```

Comment:
Do we need to wait for final decision tomorrow?

Reply:
Agreed to apply interim logic of convert "/" with "-" until we get decision to either change in frontend or backend